cmd/goal/clerk.go

```diff
@@ -142,10 +144,31 @@ var sendCmd = &cobra.Command{
 		}
 
 		client := ensureFullClient(dataDir)
+
+		// if firstValid or lastValid are nonzero, but not both, print error and exit.
+		if ((firstValid == 0) || (lastValid == 0)) && !(firstValid == 0 && lastValid == 0) {
```

Comment:
A useful default, if `firstValid` is non-zero but `lastValid` is zero, is to set `lastValid` to be `firstValid + MaxTxnLife`